themes/bootstrap3/js/facets.js

```diff
@@ -297,6 +298,10 @@ VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
     toggleSelectedFacetStyle(elem);
   }
 
+  function saveUserSelectionLastState(state) {
+    localStorage.setItem(localStorageVariableName, state);
+  }
+
   function toggleMultiFacetsSelection(enable) {
     if (typeof enable !== 'undefined') {
       isMultiFacetsSelectionActivated = enable;
```

Comment:
I think we need to return early here if `enable === isMultiFacetsSelectionActivated`. Otherwise on initialization this function can run with `enable === false` without any previous activation and do useless work and emit an event that doesn't reflect reality.

Reply:
`initMultiFacetControls` calls `toggleMultiFacetsSelection` upon initialization. if `getUserSelectionLastState()` returns false, it causes `toggleMultiFacetsSelection` to act like multi-facet selection had been active, but it hadn't been. If you do this, it won't do anything when the requested state is already active:

```
  function toggleMultiFacetsSelection(enable) {
    if (typeof enable !== 'undefined') {
      if (isMultiFacetsSelectionActivated === enabled) {
        return;
      }
      isMultiFacetsSelectionActivated = enable;
```

Reply:
Thanks for the explanation; I fixed it